reascripts/ReaSpeech/source/Widgets.lua

```diff
@@ -86,3 +86,8 @@ function Widgets.tooltip(text)
 
   ImGui.EndTooltip(ctx)
 end
+
+function Widgets.warning(text)
+  local bold_red = 0xff0000ff
```

Comment:
We might want to put stuff like this in Theme.COLORS. On a related note, at some point we might want to add light vs. dark mode. It's worth thinking about how this might affect our use of concrete color names vs. more semantic names, that could change to provide better contrast with light vs. dark backgrounds.